ctf/tests/lunatest.lua

```diff
@@ -0,0 +1,1142 @@
+-----------------------------------------------------------------------
```

Comment:
Shakedown actually uses https://github.com/Olivine-Labs/luassert
It is in "cubictest/lib" directory of shakedown.
Luassert looks much better than lunatest (18 contributors since 2013)